tfhe/src/integer/server_key/radix/mod.rs

```diff
@@ -417,6 +417,9 @@ impl ServerKey {
         ct: &mut SignedRadixCiphertext,
         num_blocks: usize,
     ) {
+        if !ct.block_carries_are_empty() {
+            self.full_propagate_parallelized(ct)
+        }
```

Comment:
do we have a test to capture the previous (I assume faulty) behavior ?

Reply:
No

Reply:
Yes the signed mul does plaintext addition to raise degree, which made some tests fail because the sign extend was not propagating carries

But there is no specially crafted test case that triggers the issue fixed

Reply:
yep all good then